docs/Tutorials/OpenAPI.md

```diff
@@ -1,9 +1,17 @@
 # OpenAPI
```

Comment:
I'm confused. 
the other tutorial documents are starting with the name of the subject like Scoping, Task.
Should I remove "# OpenAPI" and replace it with "# Overview"?

Reply:
Yup, if you have a look at the Authentication, Authorisation, or Middleware folders, you'll see an `Overview.md` with an `# Overview` title. With this OpenAPI tutorial now being under an overarching OpenAPI folder it makes this OpenAPI.md file more like an Overview.md now.

Just so it looks cleaner in the sidebar

```plain
- OpenAPI
   - Overview
```

rather than

```plain
- OpenAPI
   - OpenAPI
```

😄